_includes/team_members.html

```diff
@@ -1,7 +1,7 @@
 <div class="team-members">{% for member in members %}
   <div class="team-member">
     <div class="team-member-cell">
-      <a href="{{ site.baseurl }}/team/{{ member.name }}"><img class="team-member-thumb" src="{{ site.baseurl }}/{{ member.image }}" alt="Thumbnail photo of {{ member.full_name }}"></a>
+      <a href="{{ site.baseurl }}/team/{{ member.name }}"><img class="team-member-thumb" src="{{ site.baseurl }}/{{ member.image }}" alt="{% if member.image == 'assets/images/team/logo-18f.jpg' %}18F logo, {{ member.full_name }} photo missing{% else %}{{ member.full_name }}{% endif %}"></a>
```

Comment:
This condition should be more generic, based on `missing_team_member_img:` from `_config.yml`. Here's a suggestion:

```liquid
{% capture img_basename %}{{ member.image | split:"/" | last }}{% endcapture %}
...
{% if img_basename == site.missing_team_member_img %}...
```